test/controllers/application_controller_subclass_test.rb

```diff
@@ -62,9 +62,9 @@ class TestableResourcesControllerTest < ActionController::TestCase
       assert_equal '/realms', session[:original_uri]
     end
 
-    it "requires an account with mail" do
+    it "requires email to be set if the account has email enabled" do
```

Comment:
Does "email enabled" on this line by any chance refer to `mail_enabled` or "Mail Enabled" in the web UI? If so, may I suggest something like this?

```suggestion
    it "requires email to be set if the account has mail_enabled set to true" do
```

The wording "email enabled" is very generic and could mean a lot of things. If you're referring to the Foreman setting, it might be best to make it as clear as possible.

Reply:
Thanks, applied. FYI this won't appear anywhere in production, AFAIK it is just a test description.